test/extended/util/framework.go

```diff
@@ -938,6 +940,34 @@ func WaitForServiceAccount(c corev1client.ServiceAccountInterface, name string)
 	return wait.Poll(time.Duration(100*time.Millisecond), 3*time.Minute, waitFn)
 }
 
+// WaitForNamespaceSCCAnnotations waits until the cluster-policy-controller pods in the openshift-kube-controller-manager
+// namespace have added the SCC related annotations to the provided namespace;  NOTE: per
+// https://github.com/openshift/cluster-policy-controller/blob/master/pkg/security/controller/namespace_scc_allocation_controller.go#L74
+// the relist interval of that controller is 10 minutes, so are wait time here works off of that
+func WaitForNamespaceSCCAnnotations(c projectv1typedclient.ProjectV1Interface, name string) error {
+	waitFn := func() (bool, error) {
+		proj, err := c.Projects().Get(context.TODO(), name, metav1.GetOptions{})
```

Comment:
Think most tests are using `context.Background()`

Reply:
So I yeah I only see 

```
gmontero ~/go/src/github.com/openshift/origin/test/extended  (wait-ns-scc-annot)$ fstr.sh TODO | grep 'TODO()'
./deployments/deployments.go:			ctx1, cancel1 := context.WithTimeout(context.TODO(), deploymentChangeTimeout)
./deployments/deployments.go:			ctx2, cancel2 := context.WithTimeout(context.TODO(), deploymentChangeTimeout)
./deployments/deployments.go:			ctx3, cancel3 := context.WithTimeout(context.TODO(), deploymentChangeTimeout)
./util/framework.go:		proj, err := c.Projects().Get(context.TODO(), name, metav1.GetOptions{})
./images/oc_copy.go:		context.TODO(),
gmontero ~/go/src/github.com/openshift/origin/test/extended  (wait-ns-scc-annot)$ 
```

Conversely, context.Background() is too numerous to post here.

I'll switch - thanks.